api/src/identity-access-management/domain/usecases/update-user-details-by-admin.usecase.js

```diff
@@ -43,12 +51,26 @@ async function _checkEmailAndUsernameAreAvailable({ usersWithEmail, usersWithUse
   }
 }
 
-async function _isAddingEmailForFirstTime({ userId, email, userRepository }) {
-  const user = await userRepository.get(userId);
-  const userWithoutEmail = !user.email;
-  const userHasUsername = !!user.username;
-  const shouldChangeEmail = !!email;
+function _isAddingEmailForFirstTime({ currentUser, newEmail }) {
+  const userWithoutEmail = !currentUser.email;
+  const userHasUsername = !!currentUser.username;
+  const shouldChangeEmail = !!newEmail;
   return userWithoutEmail && userHasUsername && shouldChangeEmail;
 }
 
+async function _auditLogForEmailChanged({ currentUser, newEmail, updatedByAdminId, eventLoggingJobRepository }) {
+  if (!newEmail || newEmail === currentUser.email) return;
+
+  await eventLoggingJobRepository.performAsync(
```

Comment:
Comme dans https://github.com/1024pix/pix/blob/dev/api/src/identity-access-management/domain/usecases/update-user-email-with-validation.usecase.js#L39

```suggestion
  // Currently only used in Pix Admin, which is why app name is hard-coded for the audit log
  await eventLoggingJobRepository.performAsync(
```